buildingsync_asset_extractor/cts/cts.py

```diff
@@ -0,0 +1,82 @@
+
+import logging
+from pathlib import Path
+
+import pandas as pd
```

Comment:
I don't think this is added to the poetry file as a requirement yet...can you also make sure that you choose a version of pandas that match what is included with SEED?

Reply:
hm, I added it to the toml and its in the lock, am I missing something? seed doesn't seem to specific a pandas version, it requires https://github.com/SEED-platform/buildingid, which itself requires pandas, but neither specifics or locks a pandas versions.

I can tell you I required this new version of BAE in seed locally and it worked. 🤷🏻